static/js/redux/ui/semesterly.tsx

```diff
@@ -222,7 +222,14 @@ const Semesterly = () => {
                 <a href="/privacypolicy">Privacy</a>
               </li>
               <li className="footer-button" role="presentation">
-                <a href="mailto:contact@semester.ly?Subject=Semesterly">Contact us</a>
+                <a
+                  href=""
+                  onClick={(e) => {
+                    e.preventDefault();
+                  }}
+                >
+                  semesterly[at]jhu[dot]edu
+                </a>
```

Comment:
I'm just not sure why we'd want to do this. Do we have experience of spam from bots? 

This also looks kind of ugly on the site.
![image](https://user-images.githubusercontent.com/27985363/164076706-f8d61dcd-07d1-4ff9-85f4-90842b9545c6.png)

Reply:
Yeah, we get lots of spam from bots in the `contact` email. Perhaps, it might be auto filtered but since we're using jhu's email, it's better to minimize this possibility altogether. It looks fine to me honestly.

Reply:
I don't know where you've gotten them, but I haven't gotten a single spam email.

This design goes against the traditional footers which have contact us options. We should offer a way for them to email us natively, either by popping up a modal or redirecting to another URL if this really is a concern. I'm pretty sure bots can figure out that [at] and [dot] are the same as "@" and "."

Reply:
In the `contact@semester.ly` email as I noted before, the one that is public and only I have access to at the moment. A few hundred as it stands.

We already had a redirect to email us. That doesn't offer any protection. The ideal solution is to add ReCaptcha, but the aim here is to move emails to the new email address as quickly as possible.

Reply:
I see, spam's definitely gonna be annoying to deal with. My only concern is the UI perspective then. That screenshot is not okay.